mk/config.mk

```diff
@@ -176,6 +176,31 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary.
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA_PATHS to the paths to one or more TA ELF
```

Comment:
ok, I see the point (I had forgotten that). However it looks rather like a workaround on the config to suite the current build process.

I wonder if we could split the optee_os build in 2:
- build core only (libs + core): i.e `make core`
- build user land only (libs + devkit): i.e `make ta_devkit`

In most cases (no early TA), one could run `make all` to get everything in place.
For early TAs support: first build the user libs, then build the (early) TAs, then build core with the early TAs. This would help generating a **conf.mk** that would reflect the real configuration used to build the core binary. This is change is related to that some platform could allow the TAs to build their own 'static' libraries independently from the core configuration: enabling some debug, profiling,traces for some TAs but not for all. But maybe that should be discussed later on...